content/vaahstore/2.basics/2.setup.md

```diff
@@ -0,0 +1,310 @@
+::alert{type="warning"}
+NOTE   
+Copy the url from `Clone with SSH key` option from this link : https://git6.webreinvent.com/vaah/store-module-dev
+::  
+- The Push URL section of the Store sub-module will get autofilled with the same value as of PULL URL
+- Click on `Add Remote` present at the bottom of the panel,
+
+
+### Step-5 : Change upstream on Store Sub-Module
+::alert{type="warning"}
+NOTE   
+Whenever you create a new feature, make sure you change the upstream to GIT6,
+::
+##### Steps to Change Upstream
+- Right click on the develop or feature branch, you want to change upstream ,out of available options   
+  click on `Set Upstream` option.
+- It will ask for "What remote/branch should feature/feature-name" push to and pull from?
+- From the drop-down available, choose Git6 and click on submit button
+
+### Step-6 : Update the develop branch of Store submodule
+- Checkout to develop branch of `Store` submodule by double clicking on it,   
+- Pull the latest code by clicking on the `Pull` option from the options available on the top of the screen.
+### Step-7 : Setup the Store on the local machine
+
+ #### Follow these steps to setup the VaahStore:
+1. First you need to create database for your local machine from below provided link   
+http://localhost/phpmyadmin/   
+Follow these steps to create a database
+- Click on `New` in the Left Panel of `phpMyAdmin`
+- A new screen will be visible to you, in the `Create database` fieldset provide the database name
+- Click on `Create` button in front of it, a new database will be created and it's link will be added in   
+the sidebar of `phpMyAdmin`.   
+2. On the `GitKraken` checkout to the develop branch of the store-dev repo
+3. Go to the text editor for e.g `Php Storm`, on the top nav bar click on `File` > open, a pop screen will be visible to you,   
+   choose the directory where you cloned store-dev repo and click on `ok` button.
+4. In the root directory i.e `store-dev`, create a file named as .env (just copy the content from .env.example file)   
+     Verify these details in .env files   
+     - DB_DATABASE : it should be filled with the database name, you have just created for vaah store   
+     - DB_USERNAME : it should be filled with root   
+     - DB_PASSWORD : it should be blank
+     
+5. Now move to the browser and open the following link in your browser:   
+   http://localhost/your-directory-name/store-dev/public/vaahcms/setup, here replace your-directory-name with your directory
+
+::alert{type="warning"}
+Step 6 is for Windows users only
+::
+
+6. If you encounter PHP errors or require_once errors when trying to open your project locally,go to the terminal of `Editor`    
+   which is found in the bottom of the screen to the left side and run the command: `composer update`    
+
+::alert{type="warning"}
+Step 7, 8 and 9 are for MAC users only
+::
+7. PHP
```

Comment:
@we-suraj-k001 you can link to https://dev.to/themodernpk/install-php-for-laravel-in-macos-jkh and remove your documentation

Reply:
You should remove the documentation related to PHP and MySql setup on MacOS